docs/vault/installation.md

```diff
@@ -55,7 +37,22 @@ subkey generate --output-type json | jq '{"interbtcvault": .secretPhrase}' > key
 Please use a separate keyname and mnemonic for each client. This name determines which wallet to load on the Bitcoin full node.
 If the Vault spends funds from another wallet this may be marked as theft.
 
-### 3. Start the Vault client
+## Quickstart
+
+Setup the Vault client using docker-compose. Best if you want to quickly try out running the client.
+
+### Install docker and docker-compose
```

Comment:
It's a good idea to keep the numbers on the headings to make clear in which order steps should be performed. Can you please leave it in there?